alloc-customization-point.html

```diff
@@ -165,11 +180,106 @@ <h2>About the customization points, request for feedback/decision</h2>
+derived from <tt>true_­type</tt> if<ins>
+<br/> - </ins>the <i>qualified-id</i> <tt>T::allocator_­type</tt> is valid and denotes a type ([temp.deduct]) and 
+<tt>is_­convertible_­v&lt;Alloc, T::allocator_­type&gt; != false</tt>,<ins> or
+<br/> - if <tt>__domain_allocator_traits&lt;T, Alloc&gt;::value</tt> is <tt>true</tt></ins> 
+<br/><br/>
+<del>otherwise it</del><ins>It</ins> shall be derived from <tt>false_­type</tt><ins> otherwise</ins>. 
+A program may specialize this template to derive from <tt>true_­type</tt> for a program-defined type <tt>T</tt> that does not
+have a nested <tt>allocator_­type</tt> but nonetheless can be constructed with an allocator where either:
+<br/>- the first argument of a constructor has type allocator_­arg_­t and the second argument has type <tt>Alloc</tt> or
+<br/>- the last argument of a constructor has type <tt>Alloc</tt>.
+
+</p>
+</blockquote>
+
+<p>
+Modify [allocator.uses.construction]/p1
+<blockquote>
+ <p>
+<i>Uses-allocator construction</i> with allocator <tt>alloc</tt><ins> of type <tt>Alloc</tt></ins> and constructor arguments <tt>args...</tt> 
+refers to the construction of an object of type <tt>T</tt> such that <tt>alloc</tt><ins> or the result of invoking domain_allocator_traits&lt;class T, class Alloc&gt;::convert with 
+<tt>alloc</tt></ins> is passed to the constructor of T if T uses an allocator type compatible with <tt>alloc</tt>.
+</p></blockquote>
+</p>
+
+<p>
+Modify [allocator.uses.construction]/p2
+<blockquote><p>
+The following utility functions support <del>three</del><ins>four</ins> conventions for passing <tt>alloc</tt> to a constructor:
+<br/>- If <tt>T</tt> does not use an allocator compatible with <tt>alloc</tt>, then <tt>alloc</tt> is ignored.
+<br/><ins>- If <tt>__domain_allocator_traits&lt;T, Alloc&gt;::value</tt> is <tt>true</tt> then 
+<br/><tab2> - If T has a constructor invocable as <tt>T(allocator_­arg, domain_alloc, args...)</tt>, where domain_alloc is 
+a result of invoking <tt>__domain_allocator_traits&lt;T, Alloc&gt;::convert</tt> with <tt>alloc</tt>, then uses-allocator construction chooses this constructor form.
+<br/><tab2> - Otherwise, uses_allocator construction choses <tt>T(args..., domain_alloc)</tt>.</ins>
+<br/>- Otherwise...</p></blockquote>
+</p>
+
+<p>
+Modify [allocator.uses.construction]/p5
+<blockquote>
+ <p>
+ <i>Returns:</i> A tuple value determined as follows:<br/>
+- If <tt>uses_­allocator_­v&lt;T, Alloc&gt</tt> is <tt>false</tt> and <tt>is_­constructible_­v&lt;T, Args...&gt;</tt> is <tt>true</tt>, 
+return <tt>forward_­as_­tuple(std::forward&lt;Args&gt;(args)...)</tt>.<br/>
+<ins> - Otherwise, if <tt>__domain_allocator_traits&lt;T, Alloc&gt;::value</tt> is <tt>true</tt> then 
+<br/><tab2> - If <tt>is_­constructible_­v&lt;T, allocator_­arg_­t, const Domainalloc&, Args...&gt; is true, where Domainalloc is the return type of <tt>__domain_allocator_traits&lt;T, Alloc&gt;::convert</tt>
```

Comment:
You need to use &amp; instead of raw & here.

Reply:
fixed